server/eventlog.go

```diff
@@ -119,12 +119,18 @@ func getVerifiedCosState(coscel cel.CEL, pcrs *tpmpb.PCRs) (*pb.AttestedCosState
 	cosState.Container.EnvVars = make(map[string]string)
 	cosState.Container.OverriddenEnvVars = make(map[string]string)
 
+	seenSeparator := false
 	for _, record := range coscel.Records {
-		// ignore non COS CEL events
-		if !record.Content.IsCosTlv() {
+		// COS State only comes from the CosEventPCR
+		if record.PCR != cel.CosEventPCR {
 			continue
```

Comment:
Do we want to be strict (or at least the option to be strict) here and fail? I can imagine an attestation service implementer wanting to error on extended events into PCRs 12 or 14 since those are unexpected

Reply:
Good idea, I think we should be very strict for now, and we can relax this later.